lib/entice/logic/movement.ex

```diff
@@ -28,6 +29,9 @@ defmodule Entice.Logic.Movement do
         |> Map.put(Position, new_pos)
         |> Map.put(Movement, new_movement)
       end)
+    {:ok, %MapInstance{map: map, players: _}} = Entity.fetch_attribute(entity, MapInstance)    
+    #Can't figure out how to get eid from pid
+    Coordination.notify_all(map, {:movement_agent_updated, new_pos, entity}) #TODO: Change to use eid instead of pid for self seeking guard in seek l30
```

Comment:
Aha! This is what I was looking for after our talk the other day but all I had found was handle_change which wasn't appropriate for this case. This will simplify stuff.